woocommerce-gutenberg-products-block.php

```diff
@@ -286,3 +286,34 @@ function woocommerce_blocks_plugin_outdated_notice() {
 }
 
 add_action( 'admin_notices', 'woocommerce_blocks_plugin_outdated_notice' );
+
+/**
+ * Register the scripts. This code is basically the same we use in
+ * WordPress/block-hydration-experiments for the directives plugin with a filter
+ * to control if the runtime scripts should be enqueued.
+ */
+function wp_directives_register_scripts() {
+	// Enqueuing the directives runtime is `false` by default.
+	$should_enqueue = apply_filters( '__experimental_woocommerce_blocks_enqueue_directives_runtime', false );
+	if ( ! $should_enqueue ) {
+		return;
+	}
```

Comment:
We don't need the filter or the enqueue because `wp_enqueue_script( 'wp-directives-runtime' )` should always be called as [a dependency of another script](https://github.com/woocommerce/woocommerce-blocks/blob/748a1e9a1f12166ca014bfc4472cf5167d916e30/src/BlockTypes/SimplePriceFilter.php#L39).

Reply:
We just saw these comments after commenting ourselves on the enqueueing. Your proposal for removal of the filter makes sense, so sorry @DAreRodz for the confusion. We removed the redundant comments 😅 

However, as of the removal of these, the testing instructions for this PR are not correct anymore. So we should update them so that testers can try out these specific changes 😊